app/config/services/community.xml

```diff
@@ -145,5 +145,15 @@
             <argument>%kernel.cache_dir%</argument>
         </service>
 
+        <!-- Summary -->
+        <service id="AppBundle\Summary\SummaryFactory">
+            <argument type="service" id="cocur_slugify" />
```

Comment:
https://github.com/EnMarche/en-marche.fr/blob/master/app/AppKernel.php#L29 in https://github.com/EnMarche/en-marche.fr/commit/e9b88ab550f943ba7a09eab70a3f68524767cbcd